src/utils/utils.mjs

```diff
@@ -0,0 +1,54 @@
+export function filterSeancesByFeatures(seances, emptyAmount, hasEmptyVip, hasEmptyDouble) {
+  let filterList = [];
+  emptyAmount && filterList.push(emptySeatsFilter);
```

Comment:
can you take object, containing these params, take it keys and values - convert them to array, filter unused query parameters  out of resuting array and reduce it to form the complete query

Reply:
Did something similar